bin/rdebug-ide

```diff
@@ -49,7 +49,11 @@ EOB
 
   ENV['INSPECT_TIME_LIMIT'] = '100'
   opts.on("-t", "--time-limit LIMIT", Integer, "evaluation time limit in milliseconds (default: 100)") do |limit|
-    ENV['INSPECT_TIME_LIMIT'] = limit.to_s
+    if(limit <= 0)
```

Comment:
Is it possible to have time limit disabled and memory limit enabled?

Reply:
no